esm-samples/jsapi-oauth/oauth-callback.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
@dasa seems like with the addition of this ESM sample, maybe we can combine and remove our current oauth callback example into this one? https://github.com/Esri/jsapi-resources/tree/main/oauth

Reply:
OK, but I don't think of that as an example callback page, but more of as the canonical callback page source that users can change if they want to. We also link to it from 3.x and 4.x doc, so might be a little odd of have it in only in an example app.

Reply:
Ah yes that makes sense, I forgot about the SDK links to that directory and that we suggest users can download the callback file and host it on their server. In that case, disregard my previous comment and let's keep the `/oauth` directory, as-is.